### ISSUE TITLE
listfunds does not update all tx statuses

### ISSUE THREAD

**Issue by fixone** [ElementsProject/lightning#1307]
"version" : "v0.5.2-2016-11-21-2371-gcbe0b2d"
I have several outputs for which listfunds shows the corresponding tx as unconfirmed, but they have hundreds of confirmations in the blockchain. 
For instance f2348f107ca9b6e44d0674873021256c3aba5c4d02029dc3e7dcd84225128006 - currently 961 confirms, but listfunds shows
```
{
    "txid": "f2348f107ca9b6e44d0674873021256c3aba5c4d02029dc3e7dcd84225128006",
    "output": 1,
    "value": 38802483,
    "status": "unconfirmed"
  }
```
a complete list of outputs from listfunds
```
 lnc listfunds | jq '.outputs'
[
  {
    "txid": "3760a425946a3dd43a551964d189a6988f1e915a209009b7eb1b89cbaf03ee9a",
    "output": 1,
    "value": 7999087,
    "status": "confirmed"
  },
  {
    "txid": "c246e3540b2895f3fa24f665960b2e2680ee6eb788240244b9044aaa9b5cfb07",
    "output": 0,
    "value": 851824,
    "status": "confirmed"
  },
  {
    "txid": "4e0992dfc75367ff09bad68e6f7727a9a5535ab5923ea8c89c7c8218346890bd",
    "output": 0,
    "value": 815735,
    "status": "confirmed"
  },
  {
    "txid": "76b64bdd2a9e94bc9fbd8197d8b515be406b34e5f1a0e9ceb1c76441cc943427",
    "output": 0,
    "value": 810315,
    "status": "confirmed"
  },
  {
    "txid": "f2348f107ca9b6e44d0674873021256c3aba5c4d02029dc3e7dcd84225128006",
    "output": 1,
    "value": 38802483,
    "status": "unconfirmed"
  },
  {
    "txid": "e817b0776210f3f3872d2662196122b77aee13a2645a5265ed03eea4380149fd",
    "output": 0,
    "value": 49541,
    "status": "unconfirmed"
  },
  {
    "txid": "75ccd1c363b9af2f0076c89a6f84e6303243e266bcc1c53670e51f663b9b0df5",
    "output": 0,
    "value": 49541,
    "status": "unconfirmed"
  },
  {
    "txid": "a519ffe14e4e87cafb64bc080bbcd8acb504e2bd263c7946451e79e33c6aa9b2",
    "output": 0,
    "value": 49541,
    "status": "unconfirmed"
  },
  {
    "txid": "d8b53e329c30fc9a00cbb05f9914e21929a0303d5b7338dff6198f8aeff3299d",
    "output": 0,
    "value": 49439,
    "status": "unconfirmed"
  },
  {
    "txid": "547f6ac8322fc25c3f8741e1c21892bd0d5f2df7a2afaa3461d1ef2fa82291a5",
    "output": 0,
    "value": 633551,
    "status": "confirmed"
  },
  {
    "txid": "d08bb09b56b86c9fe3a2c1b21aeb702a8ff07ffac7e61757b2e569efb3b1fd60",
    "output": 1,
    "value": 4405813,
    "status": "unconfirmed"
  }
]
```

**Comment by ZmnSCPxj** [comment]
Possibly related: #1209 

Your version predates the merge of #1260, which fixes #1209.  Could you try to update to latest version and see if the problem goes away?

Ping @cdecker for comments.

**Comment by ZmnSCPxj** [comment]
okay, seems resolved? closing issue...
